project/LinkerdBuild.scala

```diff
@@ -83,6 +83,16 @@ object LinkerdBuild extends Base {
       .dependsOn(core)
       .withTests()
 
+
+    object Identifier {
+      val http = projectDir("linkerd/identifier/http")
+        .dependsOn(core, Router.http)
+        .withTests()
```

Comment:
tioli: it might be nicer to just keep this in the protocol project since it's a default implementation.  _especially_ since the actual identifier implementation is part of the router package and isn't actually modular.  but generally for default implementations i think it's fine to keep it part of the main protocol package.